Public/DockerTagInfo.ps1

```diff
@@ -3,4 +3,4 @@ class DockerTagInfo {
     [string] $Distro
     [string] $Version
     [string] $Tag
-}
+}
```

Comment:
```suggestion
}

```

Reply:
Line endings added to files. This file is also removed. Resolving conv.